### PR TITLE
Refactor "Create Project"

### PR DESCRIPTION
**Pull request by MeBrei**

### CODE REVIEW

File: app/src/DesignSystem/Components/RadioOption.svelte [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<script>
+  import { Icon, Title } from "../Primitives";
+  import { createEventDispatcher } from "svelte";
+  import { slide } from "svelte/transition";
+
+  export let title = "";
```
[the remark this answers is not in the document]
Comment (MeBrei) [reply app/src/DesignSystem/Components/RadioOption.svelte]: I liked the idea of indicating the "type" with the default prop, but I guess it makes conditional statements depending on them more complex.

File: app/src/DesignSystem/Components/RadioOption.svelte [hunk 2]
```diff
@@ -0,0 +1,64 @@
+<script>
+  import { Icon, Title } from "../Primitives";
+  import { createEventDispatcher } from "svelte";
```
[the remark this answers is not in the document]
Comment (MeBrei) [reply app/src/DesignSystem/Components/RadioOption.svelte]: 👍  Thanks for the suggestion!

File: app/src/Screens/CreateProject.svelte [hunk 3]
```diff
@@ -370,76 +333,61 @@
...
+          on:selected={() => (currentSelection = EXISTING)}
+          dataCy="existing-project">
+          <div slot="option-body">
+            <Text style="margin-bottom: 12px; color: var(--color-darkgray)">
+              Choose the existing repository
+            </Text>
+            <Input.Directory
+              placeholder="~/path/to/folder"
+              valid={!(validations && validations.existingRepositoryPath)}
+              validationMessage={validations && validations.existingRepositoryPath && validations.existingRepositoryPath[0]}
```
[the remark this answers is not in the document]
Comment (MeBrei) [reply app/src/Screens/CreateProject.svelte]: Yes that was one of my changes in the extract-validation commit. We would want these functions in every component that uses validations. For now maybe `PickNameStep` can live without it.

[the remark this answers is not in the document]
Comment (rudolfs) [reply app/src/Screens/CreateProject.svelte]: Ah got it, then ignore it, sorry for the noise.

**Pull request merged**